core/tests/test_utils.py

```diff
@@ -1125,6 +1129,72 @@ def save_new_default_exploration(
         exp_services.save_new_exploration(owner_id, exploration)
         return exploration
 
+    def set_interaction_for_state(self, state, interaction_id):
+        """Sets the interaction_id, sets the fully populated default
+        interaction customization arguments, and increments
+        next_content_id_index as needed.
+        """
+
+        # Since nonlocal is not available in python2, we wrap
```

Comment:
If this will work in python 3 too, then perhaps drop the first clause and just start with "We wrap ..."